pennylane_qulacs/qulacs_device.py

```diff
@@ -250,22 +270,22 @@ def apply(self, operations):
                 mapped_operation(*wires, *par).update_quantum_state(self._state)
 
     def analytic_probability(self, wires=None):
-
+        """Return the (marginal) analytic probability of each computational basis state."""
         if self._state is None:
             return None
 
         wires = wires or range(self.num_wires)
-        # 0,1 means that the qubit is observed, and 2 means no measurement.
-        measured_values = [1 if w in wires else 2 for w in range(self.num_wires)]
-        prob = self._state.get_marginal_probability(measured_values=measured_values)
```

Comment:
I get it now, qulac's ``get_marginal_prob`` function works a bit different from what we want: It returns the probability of a particular state of a subset of qubits (i.e. a float). Better to just compute it by hand like in default.qubit...